src/Symfony/Bundle/SecurityBundle/DependencyInjection/MainConfiguration.php

```diff
@@ -288,6 +288,13 @@ private function addFirewallsSection(ArrayNodeDefinition $rootNode, array $facto
                     ->scalarNode('role')->defaultValue('ROLE_ALLOWED_TO_SWITCH')->end()
                 ->end()
             ->end()
+            ->arrayNode('session_expiration')
+                ->canBeUnset()
+                ->children()
+                    ->integerNode('max_idle_time')->defaultValue(ini_get('session.gc_maxlifetime'))->min(1)->end()
```

Comment:
I'm not sure about it. If this ini option has different values for web and cli environments, it will be problematic:
if we set it at runtime, this value can vary between http requests and console commands; but if we set it at config time, the value will be read only once when the container is compiled, and the maybe the value is not the preferred one.

Anyway, ICYMI the `E_USER_WARNING` was removed and now the behavior is to log the problem with `WARN` level.

Maybe we should remove any default value and require the user to provide it.